### PR TITLE
Adjust $pidfile when run on systemd

### PR DESCRIPTION
**Pull request by arioch**

### CODE REVIEW

File: spec/classes/javagateway_spec.rb [hunk 1]
```diff
@@ -5,25 +5,27 @@
     'rspec.puppet.com'
   end
 
+  facts = {
```
Comment (bastelfreak) [inline spec/classes/javagateway_spec.rb]: can you migrate this to rspec-puppet-facts?

File: spec/default_facts.yml [hunk 2]
```diff
@@ -4,3 +4,4 @@ ipaddress: "172.16.254.254"
 is_pe: false
 macaddress: "AA:AA:AA:AA:AA:AA"
 selinux_config_mode: "disabled"
+service_provider: 'dummy'
```
Comment (bastelfreak) [inline spec/default_facts.yml]: this gets overwritten by our modulesync, please move it to default_module_facts.yml

**Comment by bastelfreak** [comment]
shouldn't /var/run be a symlink to /run? I don't see why the change is needed, could you please explain it?

**Comment by arioch** [comment]
Well this is embarrassing. You're absolutely right. 
I must've misinterpreted the output when checking.

Hence closing.

**Pull request closed without merging**